server/dive_tasks/frame_alignment.py

```diff
@@ -66,36 +56,25 @@ def _ffprobe_frame_alignment(
 
 
 def _realign_video_and_audio(
-    task: Task, output_path: str, context: Dict, manager: JobManager
-) -> str:
-    with tempfile.NamedTemporaryFile(suffix="aligned.mp4", delete=True) as temp:
-        aligned_file = temp.name
-
-    def cleanup():
-        with contextlib.supprealigned_filess(FileNotFoundError):
-            os.remove(output_path)
-
-    process_err_file = tempfile.TemporaryFile()
-    process = Popen(
-        [
-            "ffmpeg",
-            "-i",
-            output_path,
-            "-ss",
-            "0",
-            "-c:v",
-            "libx264",
-            "-preset",
-            "slow",
-            # lossless secondary encoding
-            "-crf",
-            "18",
-            "-c:a",
-            "copy",
-            aligned_file,
-        ],
-        stdout=subprocess.PIPE,
-        stderr=process_err_file,
-    )
-    stream_subprocess(process, task, context, manager, process_err_file, cleanup=cleanup)
+    task: Task, file_path: Path, context: Dict, manager: JobManager
+) -> Path:
+    aligned_file = file_path.parent / f"{file_path.name}.aligned.mp4"
+    command = [
+        "ffmpeg",
+        "-i",
+        str(file_path),
+        "-ss",
+        "0",
+        "-c:v",
+        "libx264",
+        "-preset",
+        "slow",
+        # lossless secondary encoding
+        "-crf",
+        "18",
+        "-c:a",
+        "copy",
+        aligned_file,
```

Comment:
Depending on the above comment this is really a `Path` being passed into the ffmpeg argument.